src/gam.py

```diff
@@ -7974,17 +7974,15 @@ def _getBuildingAttributes(args, body={}):
 
 def doCreateBuilding():
   cd = buildGAPIObject(u'directory')
-  body = {u'floorNames': [u'1'],
-          u'buildingId': unicode(uuid.uuid4())}
-  body = _getBuildingAttributes(sys.argv[3:], body)
+  body = _getBuildingAttributes(sys.argv[3:], {u'floorNames': [u'1',], u'buildingId': unicode(uuid.uuid4())})
```

Comment:
Let's keep this multi-line for easier readability. Let's also default body to {} in the function def.